pkg/yurtmanager/controller/loadbalancerset/loadbalancerset/localbalancerset_controller.go

```diff
@@ -0,0 +1,513 @@
+/*
+Copyright 2024 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the License);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an AS IS BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package loadbalancerset
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"reflect"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	appconfig "github.com/openyurtio/openyurt/cmd/yurt-manager/app/config"
+	"github.com/openyurtio/openyurt/cmd/yurt-manager/names"
+	"github.com/openyurtio/openyurt/pkg/apis/apps/v1beta1"
+	"github.com/openyurtio/openyurt/pkg/apis/network"
+	netv1alpha1 "github.com/openyurtio/openyurt/pkg/apis/network/v1alpha1"
+	"github.com/openyurtio/openyurt/pkg/yurtmanager/controller/loadbalancerset/loadbalancerset/config"
+)
+
+const (
+	labelManageBy = "poolservice.openyurt.io/managed-by"
+
+	poolServiceModifiedEventMsgFormat        = "PoolService %s/%s resource is manually modified, the controller will overwrite this modification"
+	poolServiceManagedConflictEventMsgFormat = "PoolService %s/%s is not managed by pool-service-controller, but the nodepool-labelselector of service %s/%s include it"
+)
+
+func init() {
+	flag.IntVar(&concurrentReconciles, "poolservice-workers", concurrentReconciles, "Max concurrent workers for PoolService controller.")
```

Comment:
please refer to pull request https://github.com/openyurtio/openyurt/pull/1906 and improve concurrentReconciles definition.

Reply:
done